<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>832</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Student Management</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxListStudent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>631</width>
      <height>228</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255, 246, 227);</string>
    </property>
    <property name="title">
     <string>List of Student: </string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignVCenter">
      <widget class="QTableWidget" name="tableWidgetListStudent">
       <property name="minimumSize">
        <size>
         <width>609</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgb(201, 233, 210);</string>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>152</number>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Code</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Age</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBoxDetails">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>631</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255, 204, 234);</string>
    </property>
    <property name="title">
     <string>Student Details:</string>
    </property>
    <widget class="QLabel" name="labelID">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>52</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditID">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 246, 227);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditCode">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 246, 227);</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCode">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>52</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Code:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditName">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 246, 227);</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>52</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAge">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 246, 227);</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelAge">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>52</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Age:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditIntro">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>180</y>
       <width>291</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 246, 227);</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelIntro">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>52</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Intro:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRemoveAvatar">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>220</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(231, 251, 230);</string>
     </property>
     <property name="text">
      <string>Remove Avatar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAvatar">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>220</y>
       <width>84</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(231, 251, 230);</string>
     </property>
     <property name="text">
      <string>Avatar</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPicture">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>221</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 246, 227);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>images/ic_picture.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxAction">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>631</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(209, 233, 246);</string>
    </property>
    <property name="title">
     <string>Action:</string>
    </property>
    <widget class="QPushButton" name="pushButtonNew">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 204, 225)</string>
     </property>
     <property name="text">
      <string>New</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/ic_new.png</normaloff>images/ic_new.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonInsert">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 204, 225)</string>
     </property>
     <property name="text">
      <string>Insert</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/ic_save.png</normaloff>images/ic_save.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonUpdate">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 204, 225)</string>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/ic_update.png</normaloff>images/ic_update.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRemove">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>30</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 204, 225)</string>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/ic_delete.png</normaloff>images/ic_delete.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
